### PR TITLE
Os-Login : Added ITSystem Test

### PR DESCRIPTION
**Pull request by athakor**
towards #5946

### CODE REVIEW

File: google-cloud-clients/google-cloud-os-login/src/test/java/com/google/cloud/oslogin/v1/it/ITSystemTest.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  private static final String REGEXP = "^\"|\"$";
+  private static final OsLoginProto.SshPublicKey SSH_PUBLIC_KEY =
+      OsLoginProto.SshPublicKey.newBuilder().build();
+  private static final Gson GSON = new Gson();
+
+  @Before
+  public void before() throws Exception {
+    osLoginServiceClient = OsLoginServiceClient.create();
+    user = getFromCredential(GOOGLE_API_CLIENT_EMAIL_ID);
+    clientId = getFromCredential(GOOGLE_API_CLIENT_ID);
```
Comment (chingor13) [inline google-cloud-clients/google-cloud-os-login/src/test/java/com/google/cloud/oslogin/v1/it/ITSystemTest.java]: nit: These could be done in a static `@BeforeAll` setup block so we only have to read them once.

File: google-cloud-clients/google-cloud-os-login/src/test/java/com/google/cloud/oslogin/v1/it/ITSystemTest.java [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  private UserName parent;
+  private String user;
+  private String clientId;
+
+  private static final String PROJECT = ServiceOptions.getDefaultProjectId();
+  private static final String GOOGLE_API_CLIENT_ID = "clientId";
+  private static final String GOOGLE_API_CLIENT_EMAIL_ID = "clientEmail";
+  private static final String GOOGLE_API_CLOUD_PLATFORM_LINK =
+      "https://www.googleapis.com/auth/cloud-platform";
+  private static final String FINGERPRINT_NAME = "test-fingerprint";
```
Comment (chingor13) [inline google-cloud-clients/google-cloud-os-login/src/test/java/com/google/cloud/oslogin/v1/it/ITSystemTest.java]: For clarity, let's name this `NON_EXISTENT_FINGERPRINT_NAME` to make it clear that we expect this one to not exist.

Comment (athakor) [reply google-cloud-clients/google-cloud-os-login/src/test/java/com/google/cloud/oslogin/v1/it/ITSystemTest.java]: done

File: google-cloud-clients/google-cloud-os-login/src/test/java/com/google/cloud/oslogin/v1/it/ITSystemTest.java (hunk 1 above)
Comment (athakor) [reply google-cloud-clients/google-cloud-os-login/src/test/java/com/google/cloud/oslogin/v1/it/ITSystemTest.java]: done

**Comment by athakor** [comment]
@chingor13 PTAL

**Pull request merged**